openmdao/core/explicitcomponent.py

```diff
@@ -129,19 +144,13 @@ def _setup_partials(self):
         # call the super version of setup_partials. This is still in the final setup.
         for out_abs, meta in self._var_abs2meta['output'].items():
 
-            # No need to FD outputs wrt other outputs
```

Comment:
Why don't we need to delete output-output keys anymore?

Reply:
That logic didn't really make sense because any changes it made would have just been overwritten at the end of the loop when the [out_abs, out_abs] entry is set to a completely new dict.